Sources/StreamVideoSwiftUI/Livestreaming/LivestreamPlayer.swift

```diff
@@ -0,0 +1,220 @@
+        self.onFullScreenStateChange = onFullScreenStateChange
+    }
+    
+    public var body: some View {
+        ZStack {
+            if viewModel.loading {
+                ProgressView()
+            } else if state.backstage {
+                Text(L10n.Call.Livestream.notStarted)
+            } else {
+                ZStack {
+                    GeometryReader { reader in
+                        if let participant = state.participants.first {
+                            VideoCallParticipantView(
+                                participant: participant,
+                                availableSize: reader.size,
+                                contentMode: .scaleAspectFit,
+                                customData: [:],
+                                call: viewModel.call
+                            )
+                            .onTapGesture {
+                                viewModel.update(controlsShown: true)
+                            }
+                            .overlay(
+                                viewModel.controlsShown ? LivestreamPlayPauseButton(
+                                    viewModel: viewModel,
+                                    controlsColor: controlsColor
+                                ) {
+                                    participant.track?.isEnabled = !viewModel.streamPaused
+                                    if !viewModel.streamPaused {
+                                        viewModel.update(controlsShown: false)
+                                    }
+                                } : nil
+                            )
+                        }
+                    }
+
+                    if viewModel.controlsShown || !viewModel.fullScreen {
+                        VStack {
+                            Spacer()
+                            HStack(spacing: 8) {
+                                LiveIndicator(controlsColor: controlsColor)
+                                if viewModel.showParticipantCount {
+                                    LivestreamParticipantsView(
+                                        participantsCount: Int(viewModel.call.state.participantCount)
+                                    )
+                                }
+                                Spacer()
+                                LivestreamButton(
+                                    imageName: !viewModel.muted ? "speaker.wave.2.fill" : "speaker.slash.fill"
+                                ) {
+                                    viewModel.toggleAudioOutput()
+                                }
+                                LivestreamButton(imageName: "viewfinder") {
+                                    viewModel.update(fullScreen: !viewModel.fullScreen)
+                                }
+                            }
+                            .padding()
+                            .background(Color.black.opacity(0.4).edgesIgnoringSafeArea(.all))
```

Comment:
Why are we using `Color` values directly instead of our own ColorPalette?

Reply:
because I forgot it 😄 Will fix it

Reply:
done